src/components/AudioCell.vue

```diff
@@ -349,6 +356,20 @@ export default defineComponent({
         URL.createObjectURL(characterInfo.iconBlob)
     );
 
+    const disableDefaultUndoRedo = (event: KeyboardEvent) => {
+      if (event.ctrlKey && (event.key == "z" || event.key == "y")) {
+        event.preventDefault();
+      }
+    };
+
+    onMounted(() => {
+      document.addEventListener("keydown", disableDefaultUndoRedo);
+    });
+
+    onUnmounted(() => {
+      document.removeEventListener("keydown", disableDefaultUndoRedo);
+    });
+
```

Comment:
良いですね！！

ここに書いてあると何度もpreventされるので、`Home.vue`に書いたほうが良いかも？
同じような目的のコードもありそうです。
https://github.com/Hiroshiba/voicevox/blob/3eef9d47b19be36f830b7770c75143a3bfbba14b/src/views/Home.vue#L205

---

Maybe it's better to write it in `Home.vue` since it will be prevented many times if it's written here?
There may be code for a similar purpose.
https://github.com/Hiroshiba/voicevox/blob/3eef9d47b19be36f830b7770c75143a3bfbba14b/src/views/Home.vue#L205